core/src/main/res/values-ar/strings.xml

```diff
@@ -37,7 +37,6 @@
   <string name="no_books_selected_toast_message">يُرجى اختيار الكتب أولًا</string>
   <string name="server_failed_message">لا يمكن بدء الخادم. يُرجى تشغيل الهوت-سبوت الخاص بك</string>
   <string name="server_failed_toast_message">لا يمكن بدء الخادم.</string>
-  <string name="server_started_successfully_toast_message">بدأ الخادم بنجاح.</string>
```

Comment:
You deleted these strings?

Reply:
Sorry, it happened due to git conflicts. Have added it back.